mods/boats/init.lua

```diff
@@ -224,10 +224,10 @@ minetest.register_craftitem("boats:boat", {
 
 	on_place = function(itemstack, placer, pointed_thing)
```

Comment:
Are you kidding me? It definitely does. `pointed_thing` is being used later, and this would crash then.